shtdlib.sh

```diff
@@ -138,9 +141,28 @@ elif [ "${os_family}" == 'Alpine' ]; then
     os_name='alpine'
 fi
 
-# Store local IP addresses (not localhost)
+# Gets local IP addresses (excluding localhost)
+function get_local_ip_addresses {
+    local ip_addrs
+    ip_addrs=$( (
```

Comment:
I don't think we need a space in there, do you think it's more readable this way or without?

Reply:
Removed the space; agreed that it is more readable without.